cluster/local/integration_tests.sh

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env bash
```

Comment:
@stevendborrelli can we remove this integration tests file from this PR? Looks like it's included in #49 also, so let's remove it from here and keep this PR as only the provider checklist markdown file.

Reply:
It's been removed.